organization.go

```diff
@@ -0,0 +1,33 @@
+package clerk
+
+import "encoding/json"
+
+type Organization struct {
+	APIResource
+	Object                  string          `json:"object"`
+	ID                      string          `json:"id"`
+	Name                    string          `json:"name"`
+	Slug                    string          `json:"slug"`
+	ImageURL                *string         `json:"image_url"`
+	HasImage                bool            `json:"has_image"`
+	MembersCount            *int64          `json:"members_count,omitempty"`
+	PendingInvitationsCount *int64          `json:"pending_invitations_count,omitempty"`
+	MaxAllowedMemberships   int64           `json:"max_allowed_memberships"`
+	AdminDeleteEnabled      bool            `json:"admin_delete_enabled"`
+	PublicMetadata          json.RawMessage `json:"public_metadata"`
+	PrivateMetadata         json.RawMessage `json:"private_metadata"`
+	CreatedBy               string          `json:"created_by"`
+	CreatedAt               int64           `json:"created_at"`
+	UpdatedAt               int64           `json:"updated_at"`
+
+	// LogoURL is the URL of the organization's logo.
+	//
+	// Deprecated: Use ImageURL instead.
+	LogoURL *string `json:"logo_url,omitempty"`
```

Comment:
❓ As this is a deprecated property, i believe we should stop using on the new major version

Reply:
I have the same opinion. Same for deprecated endpoints.

However, we had a discussion with @alex-ntousias and he suggested that we keep everything that we support in v1 so that we make the transition to v2 easier.

I guess for struct fields with a clear alternative things are easier. I'll drop the field.